src/modules/popUp.js

```diff
@@ -23,30 +26,59 @@ export default class PopUp {
     container.querySelector('i').addEventListener('click', () => {
       container.remove();
     });
+
+    if (this.data.type === 'Recipe') {
+      const [user, comment] = container.querySelectorAll('input');
+      const submit = container.querySelector('button');
+
+      submit.addEventListener('click', async () => {
+        if (user.value === '' || comment.value === '') return;
+        const inputComment = { username: user.value, comment: comment.value };
+        const [day, month, year] = new Date().toLocaleDateString().split('/');
+        const li = document.createElement('li');
+        li.innerHTML = PopUp.displayComment(
+          {
+            ...inputComment,
+            creation_date:
+              `${year}-${month < 10 ? '0' : ''}${month}-${day < 10 ? '0' : ''}${day}`,
+          },
+        );
+        await InvolvementAPI.addComment(inputComment, this.data.idMeal);
+        container.querySelector('ul').appendChild(li);
+        this.commentCount += 1;
+        container.querySelector('.recipes__popup_comment-count').innerHTML = `Comments (${this.data.comments.length + this.commentCount})`;
+        [user.value, comment.value] = ['', ''];
+      });
+    }
```

Comment:
**Changes Required**
It would be good to separate this into multiple functions, to follow the single responsibility principle.